draft-ietf-lsr-isis-fast-flooding.xml

```diff
@@ -514,7 +514,7 @@
 			<t>Congestion control creates multiple interacting control loops between multiple transmitters and multiple receivers to prevent the transmitters  from overwhelming the overall network. For an IS-IS adjacency, the network between two IS-IS neighbors is relatively limited in scope and consist of a link that is typically over-sized compared to the capability of the IS-IS speakers, but may also include components inside both routers such as a switching fabric, line card CPU, and forwarding plane buffers that may experience congestion. These resources may be shared across multiple IS-IS adjacencies for the system and it is the responsibility of congestion control to ensure that these are shared reasonably.</t>
 			<t>Reliability provides loss detection and recovery. IS-IS already has mechanisms to ensure the reliable transmission of LSPs. This is not changed by this document.</t>
 
-			<t>The following two sections provide examples of Flow and/or Congestion control algorithms as examples that may be implemented by taking advantage of the extensions defined in this document. They are non-normative. The IS-IS extensions defined in  <xref target="FloodingTLV"/> and  <xref target="Receiver"/> are generic and are designed to support different sender-side algorithms. A sender can unilaterally choose a different algorithm to use.</t>
+			<t>The following two sections provide two Flow and/or Congestion control algorithms that may be implemented by taking advantage of the extensions defined in this document. The signal that theses IS-IS extensions defined in  <xref target="FloodingTLV"/> and  <xref target="Receiver"/>provides are generic and are designed to support different sender-side algorithms. A sender can unilaterally choose a different algorithm to use.</t>

```

Comment:
s/theses/these/ ?

Reply:
s/
`and  <xref target="Receiver"/>provides ` /
`and <xref target="Receiver"/> provide` /

Reply:
thanks. Done.